_pages/install/site.adoc

```diff
@@ -0,0 +1,348 @@
+---
+layout: docs-base
+html-class: docs-page
+title: Metanorma Site Generation
+---
+
+= Metanorma Site Generation (`metanorma site`)
+
+== Introduction
+
+The `site` subcommand enables you to compile and publish multiple Metanorma
+documents as a cohesive document site.
+This functionality is particularly useful for:
+
+* Multiple documents belonging to the same series
+* Single documents published to multiple venues
+* Multiple parts of a document collection
+
+== Commands
+
+=== `metanorma site generate`
+
+[source,sh]
+----
+metanorma site generate [OPTIONS...] [SOURCE_PATH]
+----
+
+If no source path is provided, the current directory is used.
+
+=== Options
```

Comment:
1. Place the Options in the same section. 
1. Use a definition list for every option.